ios/IoReactNativeCrypto.swift

```diff
@@ -200,6 +200,74 @@ class IoReactNativeCrypto: NSObject {
       )
     }
   }
+
+  @objc
+  func unpackBerEncodedASN1(
+    _ signature: String,
+    coordinateOctetLength: Int,
+    resolver resolve: RCTPromiseResolveBlock,
+    rejecter reject: RCTPromiseRejectBlock
+  ) {
+    do {
+      let signatureData = Data(base64Encoded: signature)
+      let ecSignatureTLV = [UInt8](signatureData!)
+      let ecSignature = try ecSignatureTLV.read(.sequence)
+      let varlenR = try Data(ecSignature.read(.integer))
+      let varlenS = try Data(ecSignature.skip(.integer).read(.integer))
+      let fixlenR = Asn1IntegerConversion.toRaw(varlenR, of: coordinateOctetLength)
+      let fixlenS = Asn1IntegerConversion.toRaw(varlenS, of: coordinateOctetLength)
+      let sign = fixlenR + fixlenS
+      resolve(String(
+        decoding: Data(sign).base64EncodedData(),
+        as: UTF8.self
+       ))
+    }
+    catch {
+      reject("Error", "\(error)", error);
```

Comment:
What do you think to integrate with our standard way to reject by extending our `ModuleException` and provide a specific exception type also exporting it to RN?